api/src/neo4j/queries/gene.js

```diff
@@ -46,5 +47,94 @@ RETURN apoc.map.mergeList(COLLECT(value.data)) as gene
   return { ...gene, externalDbs: reformatExternalDbs(gene.externalDbs) };
 };
 
+const getGenesForHPA = async () => {
+  const statement = `
+MATCH (g:Gene:HumanGem)-[:V1_3_0]-(r:Reaction)-[:V1_3_0]-(s:Subsystem)-[:V1_3_0]-(ss:SubsystemState)
```

Comment:
Even though the model is static (which is as expected), it would be highly desirable for the version to automatically take the latest. We should not need to remember to bump up the version here every time we update the model in `data-files`.
Note: this version is used in multiple places in this file.